config/settings.js

```diff
@@ -27,11 +35,19 @@
  */
 
 window.spinnakerSettings = {
+  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
```

Comment:
We could potentially special case this by having the script that rewrites
this file derive the value for this based on the provider.enabled values it
encounters.

On Tue, Jan 26, 2016 at 4:59 PM, Chris Berry <notifications@github.com>
wrote:

> In config/settings.js
> <https://github.com/spinnaker/spinnaker/pull/702#discussion_r50907529>:
>
> > @@ -27,11 +35,19 @@
> >   */
> >
> >  window.spinnakerSettings = {
> > +  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
>
> That would be nice - not sure how difficult it would be, but definitely
> would be nice.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spinnaker/spinnaker/pull/702/files#r50907529>.
>